Npm reorg

afustePTC opened the pull request:
Created a package.json and package-lock.json to the root of the repo.
And removed all other package.json files from the hardware interfaces folders.
I was working on the lego-boost kineticAR interface but I removed that and will add it to another branch

hobinjk-ptc commented:
The old `npm run build` on master succeeds in tools/motion so it's probably a funky issue with paths

afustePTC commented:
> The old `npm run build` on master succeeds in tools/motion so it's probably a funky issue with paths

I removed some files that were still there and added babel (I had removed it before cause I thought we didn't need it but apparently we do). can you try again? thanks!
